utils/releaser/src/ReleaseWorker/Release/CreateAndPushGitTagsExceptProjectBaseReleaseWorker.php

```diff
@@ -0,0 +1,73 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Shopsys\Releaser\ReleaseWorker\Release;
+
+use PharIo\Version\Version;
+use Shopsys\Releaser\ReleaseWorker\AbstractShopsysReleaseWorker;
+use Shopsys\Releaser\Stage;
+
+final class CreateAndPushGitTagsExceptProjectBaseReleaseWorker extends AbstractShopsysReleaseWorker
+{
+    private const PACKAGES_WITHOUT_MONOREPO_AND_PROJECT_BASE = [
+        'http-smoke-testing',
+        'coding-standards',
+        'read-model',
+        'product-feed-google',
+        'product-feed-zbozi',
+        'product-feed-heureka-delivery',
+        'product-feed-heureka',
+        'plugin-interface',
+        'framework',
+        'form-types-bundle',
+        'migrations',
+        'google-cloud-bundle',
+    ];
+
+    /**
+     * @param \PharIo\Version\Version $version
+     * @return string
+     */
+    public function getDescription(Version $version): string
+    {
+        return 'Create and push git tags for packages excluding monorepo and project-base';
+    }
+
+    /**
+     * Higher first
+     * @return int
+     */
+    public function getPriority(): int
+    {
+        return 640;
+    }
+
+    /**
+     * @param \PharIo\Version\Version $version
+     */
+    public function work(Version $version): void
+    {
+        $versionString = $version->getVersionString();
+
+        $this->processRunner->run('mkdir temp');
+        foreach (self::PACKAGES_WITHOUT_MONOREPO_AND_PROJECT_BASE as $packageName) {
+            $this->processRunner->run('cd temp && git clone https://github.com/shopsys/' . $packageName . '.git');
+            $this->processRunner->run('cd temp/' . $packageName . ' && git checkout ' . $this->initialBranchName . ' && git tag ' . $versionString);
+            $this->processRunner->run('cd temp/' . $packageName . ' && git push origin ' . $versionString);
+        }
+
+        $this->symfonyStyle->note('Wait for packagist to get new versions of all packages excluding monorepo and project-base');
```

Comment:
I don't know  about this one.. do we have this logic somewhere in releaser?

Reply:
well, depends on what you mean by "this logic" :grin: check of the packagist is implemented in the mentioned `CheckPackagesOnPackagistReleaseWorker`, the while loops are on our wishlist, I think it would be really helpful to implement it here, or not? :slightly_smiling_face: